CHANGELOG.md

```diff
@@ -1,14 +1,4 @@
-## 2.10.0 (Unreleased)
-## 2.9.0 (June 29, 2020)
```

Comment:
I wouldn't revert the changelog like this, would just add a comment as to what happened in 2.9.0, we can pull it from the registry, etc, but you wouldn't want to reuse the number anyway.

Reply:
Makes sense.  Ideally this merges and releases as `v2.9.1`, a complete revert of `v2.9.0`.  Will ensure the CHANGELOG reflects this history.

Reply:
@paultyng 24418cc addresses this concern.